packages/react-instantsearch/src/widgets/Autocomplete.tsx

```diff
@@ -195,6 +211,45 @@ export function EXPERIMENTAL_Autocomplete<TItem extends BaseHit = BaseHit>({
     });
   }
 
+  const recentSearchConfig = showRecent
```

Comment:
Not completely familiar with the conventions, but shouldn't classNames assignment be done in the ui-components and just be passed on from the widget?